app/models/user.rb

```diff
@@ -14,6 +14,8 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
 
+  before_create :generate_auth_token
```

Comment:
why do user need token on create if it may expire after 1 month..?
it's better to allow user to (re-)generate API token within profile page

Reply:
there is an option to (re-)generate token from profile page, and it is also set on user create